### ISSUE TITLE
importing models does not import scale from blender

### ISSUE THREAD

**Issue by droqen** [AmbientRun/Ambient#848]
Image of model as viewed in 'Windows 3D Viewer' as well as Blender, Godot:
![image](https://github.com/AmbientRun/Ambient/assets/5104632/a9904ce4-9b77-4684-a6cd-87a0a20d079c)

Image of model imported into Ambient via model_from_url or prefab_from_url:
![image](https://github.com/AmbientRun/Ambient/assets/5104632/35fc86b8-5808-42de-8cb6-c9c4e5fa10db)

In Blender, the problem is clear: The model's vertices are being imported, but not its transform data.

See import:
![image](https://github.com/AmbientRun/Ambient/assets/5104632/623774c3-c230-4608-a068-10fb1e4dcb9f)

See import with scale set to 1, 1, 1:
![image](https://github.com/AmbientRun/Ambient/assets/5104632/cc405bb8-e7fb-4039-a1ec-62755dbf87b2)

**Comment by kevzettler** [comment]
can share the model for testing or you can confirm behavior when you import the model in :
https://sandbox.babylonjs.com/

and 

https://gltf-viewer.donmccurdy.com/

**Comment by droqen** [comment]
Good point. I've attached a zipped **really_tall_lumpy_wall.glb** file which renders in the 'correct' way in both those importers but not Ambient:

[really_tall_lumpy_wall.zip](https://github.com/AmbientRun/Ambient/files/12590527/really_tall_lumpy_wall.zip)

And this is a version with transformations applied, **tall_noscale.glb** (note the texture looks a bit different, it's weird), which works the same in everything:

[tall_noscale.zip](https://github.com/AmbientRun/Ambient/files/12590543/tall_noscale.zip)

**Comment by kevzettler** [comment]
thanks @droqen sounds like a bug
